Explorer/Assets/DCL/Multiplayer/Connections/Rooms/InteriorRoom.cs

```diff
@@ -0,0 +1,147 @@
+        {
+            ConnectionQualityChanged?.Invoke(quality, participant);
+        }
+
+        private void RoomOnTrackUnmuted(TrackPublication publication, Participant participant)
+        {
+            TrackUnmuted?.Invoke(publication, participant);
+        }
+
+        private void RoomOnTrackMuted(TrackPublication publication, Participant participant)
+        {
+            TrackMuted?.Invoke(publication, participant);
+        }
+
+        private void RoomOnTrackUnsubscribed(ITrack track, TrackPublication publication, Participant participant)
+        {
+            TrackUnsubscribed?.Invoke(track, publication, participant);
+        }
+
+        private void RoomOnTrackSubscribed(ITrack track, TrackPublication publication, Participant participant)
+        {
+            TrackSubscribed?.Invoke(track, publication, participant);
+        }
+
+        private void RoomOnTrackUnpublished(TrackPublication publication, Participant participant)
+        {
+            TrackUnpublished?.Invoke(publication, participant);
+        }
+
+        private void RoomOnTrackPublished(TrackPublication publication, Participant participant)
+        {
+            TrackPublished?.Invoke(publication, participant);
+        }
+
+        private void RoomOnLocalTrackUnpublished(TrackPublication publication, Participant participant)
+        {
+            LocalTrackUnpublished?.Invoke(publication, participant);
+        }
+
+        private void RoomOnLocalTrackPublished(TrackPublication publication, Participant participant)
+        {
+            LocalTrackPublished?.Invoke(publication, participant);
+        }
+
+        private void RoomOnRoomMetadataChanged(string metadata)
+        {
+            RoomMetadataChanged?.Invoke(metadata);
+        }
+
+        public event Room.MetaDelegate? RoomMetadataChanged;
+        public event LocalPublishDelegate? LocalTrackPublished;
+        public event LocalPublishDelegate? LocalTrackUnpublished;
+        public event PublishDelegate? TrackPublished;
+        public event PublishDelegate? TrackUnpublished;
+        public event SubscribeDelegate? TrackSubscribed;
+        public event SubscribeDelegate? TrackUnsubscribed;
+        public event MuteDelegate? TrackMuted;
+        public event MuteDelegate? TrackUnmuted;
+        public event ConnectionQualityChangeDelegate? ConnectionQualityChanged;
+        public event ConnectionStateChangeDelegate? ConnectionStateChanged;
+        public event ConnectionDelegate? ConnectionUpdated;
+
+        public Task<bool> Connect(string url, string authToken, CancellationToken cancelToken) =>
+            assigned.EnsureAssigned().Connect(url, authToken, cancelToken);
+
+        public void Disconnect() =>
+            assigned.EnsureAssigned().Disconnect();
+
+        public IActiveSpeakers ActiveSpeakers => activeSpeakers;
+        public IParticipantsHub Participants => participants;
+        public IDataPipe DataPipe => dataPipe;
```

Comment:
please apply DCL code style for proper class members ordering

Reply:
send the link to the code style file please, I'll link it to Rider